backup-cronjob.yaml

```diff
@@ -12,7 +12,7 @@ spec:
           containers:
           - command:
             - /bin/sh
-            - /scripts/backup.sh
+            - /usr/local/bin/backup.sh
             image: ghcr.io/adfinis-adsygroup/openshift-etcd-backup
```

Comment:
Shouldn't the image originate from quay.io, as quay.io has to be allowed already to deploy OpenShift?

Reply:
yes that makes total sense!

Reply:
imo ghcr.io is fine as default, quay.io is really only offered for finicky customers.

using ghcr.io (our default oss registry) is preferable if possible.

Reply:
done!